sdk/README.md

```diff
@@ -34,7 +34,7 @@ a basic sequential pipeline.
 4. Compile the sample pipeline:
 
     - `cd sdk/samples`  
-    - `dsl-compile-tekton --py ./parallel_join.py --output pipeline.yaml`
```

Comment:
removing the `./` notation since we are not using the Python script as an executable here